app/master/serial_request_handler.py

```diff
@@ -64,10 +48,7 @@ def _compute_subjobs_for_build(self, build_id, job_config, project_type):
         :type project_type: project_type.project_type.ProjectType
         :rtype: list[Subjob]
         """
-        try:
-            atoms_list = job_config.atomizer.atomize_in_project(project_type)
-        except AtomizerError as ex:
-            raise BuildProjectError('Build failed during atomization.') from ex
+        atoms_list = job_config.atomizer.atomize_in_project(project_type)
```

Comment:
I moved the definition of BuildProjectError to Build.  I thought about defining a new type of error to be raised here, or importing BuildProjectError, but then I wasn't sure if it's necessary to transform an AtomizerError to a BuildProjectError.  Maybe I'm missing something though.